src/components/HelpResources/HelpResources.js

```diff
@@ -1,12 +1,21 @@
 import React from "react";
+import { GridContainer, Grid } from "@trussworks/react-uswds";
+import { HelpResourcesCards } from "../HelpResourcesCards/HelpResourcesCards";
 
 export const HelpResources = () => {
   return (
-    <div>
-      <h1 style={{ textAlign: "left" }}>Help & Resources</h1>
-      <p style={{ textAlign: "left" }}>
-        What to find in Help & Resources sections
-      </p>
-    </div>
+    <GridContainer>
+      <Grid row gap>
+        <Grid desktop={{ col: 8 }} className="text-left">
+          <h1 style={{ textAlign: "left" }}>Help & Resources</h1>
+          <p style={{ textAlign: "left" }}>
+            What to find in Help & Resources sections
```

Comment:
Because this component is so simple, I think it might be worthwhile to put all the cards in the HelpResources component instead of splitting it in two. It made sense for the HomePage because there's a lot of content in there besides the two cards, but in this case, I think it'd be ok to have everything in one component.